app/src/main/java/com/example/roomies/SettingsFragment.java

```diff
@@ -73,6 +76,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         Bundle bundle = this.getArguments();
         circle = bundle.getParcelable("circle");
+        userCircle = null;
+        getCircle();
```

Comment:
Can we update the `getCircle()` to returns a circle object, instead of having it as a void functions? Conventionally, getXXX method should returns something.

Reply:
I'm not sure if there is a way to use await? But also I don't think I can use await on the done() function that I am overriding. Plus, done() wouldn't be able to return anything anyways.